etabotsite/etabotapp/views.py

```diff
@@ -416,6 +374,7 @@ def post(self, request, format=None):
         # includes TMS credentials
         # threads = []
         global_params = post_data.get('params', {})
+        global_params['oauth_obj'] = oauth
```

Comment:
I'm wondering if tmrw we are supporting oauth for a different platform say trello board, which oauth should we set the `global_params` to？